zookeeper-server/pom.xml

```diff
@@ -167,6 +167,12 @@
       <artifactId>guava</artifactId>
       <version>27.0.1-jre</version>
     </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.6</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
Do we need a new dependency just for using `FileUtils.listFiles()` and `FileUtils.deleteDirectory()`? Can we implement this ourselves?

Reply:
I think your concern is valid. I updated the code to remove usage of FileUtils.